crates/sui-indexer/src/metrics.rs

```diff
@@ -3,17 +3,19 @@
 
 use std::collections::HashMap;
 use std::net::SocketAddr;
+use std::time::{Duration, SystemTime, UNIX_EPOCH};
 
 use axum::{extract::Extension, http::StatusCode, routing::get, Router};
+use downcast::Any;
```

Comment:
We shouldn't need this, and instead should be able to get the name of the histogram itself